projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectCode.java

```diff
@@ -0,0 +1,157 @@
+package life.qbic.projectmanagement.domain.project;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Random;
+import java.util.function.Supplier;
+
+/**
+ * QBiC Project Code
+ *
+ * Represents a project code, that is usually communicated with customers and internally
+ * to reference a project.
+ *
+ * @since 1.0.0
+ */
+public class ProjectCode {
+
+  public static final String[] BLACKLIST = new String[]{"FUCK", "SHIT"};
+
+  private final String value;
+
+  private static final int LENGTH = 5;
+
+  private static final String PREFIX = "Q";
+
+  /**
+   * Creates a random project code containing of letters from the English alphabet and natural
+   * numbers.
```

Comment:
and numbers? I thought its just letters for the Project Code. Numbers are included for e.g sample codes

Reply:
Actually it is numbers and letters :)